pkgs/development/tools/gazou/default.nix

```diff
@@ -0,0 +1,28 @@
+# gazou.nix
+{ stdenv, fetchFromGitHub, cmake, pkgconfig , lib , tesseract , leptonica , qt5 }:
+
+stdenv.mkDerivation rec {
+  pname = "gazou";
+  version = "0.001-2023-08-18";
```

Comment:
Latest release seems to be the 0.2.1, please use that instead

Reply:
Correct Adjustment Verification:
version = "0.2.1";

Reply:
Addressed.